login/index.js

```diff
@@ -0,0 +1,72 @@
+const inputEmail = document.querySelector('#login-email');
+const inputPassword = document.querySelector('#login-password');
+const loginButton = document.querySelector('#login-button');
+
+inputEmail.addEventListener('focusout', noInput);
+inputEmail.addEventListener('focusout', emailCheck);
+inputEmail.addEventListener('keyup', enterKey);
+
+inputPassword.addEventListener('focusout', noInput);
+inputPassword.addEventListener('keyup', enterKey);
+
+loginButton.addEventListener('click', correctInput);
+
+
+// input 없을 때 에러 메세지
+function noInput(e) {
```

Comment:
`noInput` 함수명이 직관적이지 않은 것 같아요. 차라리 focusout시에 처리하는 핸들러 함수이니 함수명도 거기에 맞춰서 수정하면 직관적일 수 있을 것 같습니다!
```suggestion
function handleFocusOut(e) {
```